File: 04300/04340/4344.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int cnt = Integer.parseInt(br.readLine());

        for (int i = 0; i < cnt; i++) {
            StringTokenizer st = new StringTokenizer(br.readLine(), " ");

            int students = Integer.parseInt(st.nextToken());
            int[] scores = new int[students];
            float sum = 0;

            for (int j = 0; j < students; j++) {
                scores[j] = Integer.parseInt(st.nextToken());
                sum += scores[j];
            }

            float avg = sum / students;
            int count = 0;

            for (int j = 0; j < students; j++) {
                if (scores[j] > avg) {
                    count++;
                }
            }

            float percentage = (float) count / students * 100;
            System.out.printf("%.3f%%\n", percentage);
        }
    }
}

File: 04300/04340/4344.c
#include<stdio.h>

int main() {
	int n, m;
	scanf("%d", &n);

	for (int i = 0; i < n; i++) {
		scanf("%d", &m);
		int arr[1000], sum = 0;
		double avg;
		for (int j = 0; j < m; j++) {
			scanf("%d", &arr[j]);
			sum += arr[j];
		}
		avg = (double)sum / m;
		int cnt = 0;
		for (int j = 0; j < m; j++) {
			if (avg < arr[j])
				cnt++;
		}
		printf("%.3f%%\n", (double)cnt * 100 / m);
	}
	return 0;
}